src/main/webapp/app/overview/course-conversations/layout/conversation-selection-sidebar/conversation-selection-sidebar.component.html

```diff
@@ -32,26 +32,25 @@ <h3 class="card-title">
                 </div>
                 <div class="scrollbar wrapper-scroll-y">
                     <!-- Starred -->
-                    <div [hidden]="!displayedStarredConversations?.length" class="mb-2">
```

Comment:
Why you removed the hidden here?

Reply:
Because this functionality is now also handled by the jhi-conversation-sidebar-section component